tests/cross/src/test/scala/tests/pc/CompletionSnippetSuite.scala

```diff
@@ -439,4 +439,29 @@ class CompletionSnippetSuite extends BaseCompletionSuite {
     includeDetail = true
   )
 
+  // https://github.com/scalameta/metals/issues/4004
+  checkSnippet(
+    "issu-4004".tag(IgnoreScala2),
```

Comment:
Super nitpick, ignore if you will not be changing anything else.

```suggestion
    "issue-4004".tag(IgnoreScala2),
```